includes/shortcodes.php

```diff
@@ -272,20 +272,29 @@ function edd_downloads_query( $atts, $content = null ) {
 		'thumbnails'       => 'true',
 		'orderby'          => 'post_date',
 		'order'            => 'DESC',
-		'ids'              => ''
+		'ids'              => '',
+		'pagination'       => 'true'
 	), $atts, 'downloads' );
 
 	$query = array(
 		'post_type'      => 'download',
-		'posts_per_page' => (int) $atts['number'],
 		'orderby'        => $atts['orderby'],
 		'order'          => $atts['order']
 	);
 
-	if ( $query['posts_per_page'] < -1 ) {
-		$query['posts_per_page'] = abs( $query['posts_per_page'] );
+	if(  'true' === $atts['pagination']  ) {
+
+		$query['posts_per_page'] = (int) $atts['number'];
+
+		if ( $query['posts_per_page'] < -1 ) {
```

Comment:
`$query['posts_per_page'] < -1` should be `$query['posts_per_page'] < 0`. By being `< -1` would allow a value of `-1`.